common/display/display-en.html

```diff
@@ -0,0 +1,969 @@
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.opct-20 </code>class working example</h4><div class="well">
+	<div class="opct-20 well">
+		This element has an opacity of 0.2 (20%).
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>opct-20</strong> well"&gt;
+		This element has an opacity of 0.2 (20%).
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.opct-90 </code>class working example</h4><div class="well">
+	<div class="opct-90 well">
+		This element has an opacity of 0.9 (90%).
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>opct-90</strong> well"&gt;
+		This element has an opacity of 0.9 (90%).
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.opct-100 </code>class working example</h4><div class="well">
+	<div class="opct-100 well">
+		This element has an opacity of 1 (100%).
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>opct-100</strong> well"&gt;
+		This element has an opacity of 1 (100%).
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3 id="wet-toggling-content">WET - Toggling content</h3>
+
+<h4>WET-BOEW <code>.nojs-show </code>class working example</h4><div class="well">
+	<div class="nojs-show well">
+		This element is initially hidden, but it will be shown when JavaScript is disabled.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>nojs-show</strong> well"&gt;
+		This element is initially hidden, but it will be shown when JavaScript is disabled.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.nojs-hide </code>class working example</h4><div class="well">
+	<div class="nojs-hide well">
+		This element is initially visible, but it will be hidden when JavaScript is disabled.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>nojs-hide</strong> well"&gt;
+		This element is initially visible, but it will be hidden when JavaScript is disabled.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.lbx-hide-gal </code>class working example</h4><div class="well">
+	<div class="lbx-hide-gal well">
+		This element is hidden as part of a lightbox gallery.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>lbx-hide-gal</strong> well"&gt;
+		This element is hidden as part of a lightbox gallery.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.mfp-hide </code>class working example</h4><div class="well">
+	<div class="mfp-hide well">
+		This element is hidden as part of a Magnific Popup modal.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>mfp-hide</strong> well"&gt;
+		This element is hidden as part of a Magnific Popup modal.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
```

Comment:
`mfp-hide` + `lbx-hide-fal` example can be removed

Reply:
Fixed